json/src/test/java/com/basistech/rosette/dm/json/plain/JsonTest.java

```diff
@@ -645,6 +646,7 @@ public void roundTrip() throws Exception {
         assertNotNull(entityList);
         assertEquals(1, entityList.size());
         Entity en = entityList.get(0);
+        assertEquals(0.4, en.getSalience(), 0.0001); // just make sure the salience field works all around.
```

Comment:
Because I forgot to deal with the array test when I did Entity in the first place, so there's no place to put it -- and because Entity isn't specially customized for arrays.